### PR TITLE
FE: Redesign menu

### LINKED ISSUE #195: FE: Redesign menu
https://www.figma.com/file/IBvMpXfwbwd8d5IeOZXslK/Kafbat-UI?type=design&node-id=2535-8274&mode=design&t=LkEmI6Ei5Mdo5aQI-0

### PR DESCRIPTION
**Pull request by Leshe4ka**
Resolve #195 

- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [ ] My changes generate no new warnings (e.g. Sonar is happy)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/kafbat/kafka-ui/blob/main/.github/CONTRIBUTING.md) and [Code of Conduct](https://github.com/kafbat/kafka-ui/blob/main/.github/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by Leshe4ka** [comment]
review pls

File: frontend/src/lib/hooks/useScreenSize.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  const checkScreenSize = () => {
+    setScreenSize({
+      isLarge: window.innerWidth > M,
+      isMedium: window.innerWidth >= S && window.innerWidth <= M,
+    });
+  };
+
+  useEffect(() => {
+    window.addEventListener('resize', checkScreenSize);
```
Comment (Mgrdich) [inline frontend/src/lib/hooks/useScreenSize.ts:22]: maybe we can write the function exactly in the `useEffect` so we don't have to lie about the dependencies for the `useEffect`.

**Review by Mgrdich — APPROVED** [review]

**Review by Haarolean — CHANGES_REQUESTED** [review]
Quite broken with the light theme:
image

**Comment by Leshe4ka** [comment]
> Quite broken with the light theme: image

Resolved

**Review by Haarolean — APPROVED** [review]
Committed a fix for the status icon size, the rest seems to be good, thanks

**Pull request merged**